website/content/docs/nia/cli/index.mdx

```diff
@@ -19,28 +21,6 @@ $ consul-terraform-sync -config-file=config.hcl
 
 To view a list of available flags, use the `-help` or `-h` flag.
 
-### Modes
-
-CTS can be run as a daemon in different modes.
-
-#### Long-running Mode
-
-Flag: none
-
-Behavior: This is the default mode in which CTS passes through a once-mode phase and then turns into a long running process. During the once-mode phase, the daemon will exit with a non-zero status if it encounters an error. After successfully passing through once-mode phase, it will begin a long-running process in which errors are logged and exiting is not expected behavior. Once beginning the long-running process, the daemon serves any API and command requests.
-
-Usage: Intended to be run as a long running process after running once-mode successfully for given configuration and tasks.
-
-#### Inspect Mode
-
-Flag: `-inspect`
-
-Behavior: CTS will display the proposed state changes for all tasks once and exit. No changes are applied in this mode. On encountering an error before completing, CTS will exit with a non-zero status.
-
-Usage: Intended to be run before daemon-mode in order to confirm configuration is accurate and tasks would update network infrastructure as expected.
-
----
-
```

Comment:
Looks like you can remove more? - including Once Mode (up to the Command section)